src/components/cart.js

```diff
@@ -132,6 +132,14 @@ export default class Cart extends Component {
     }
   }
 
+  createCheckout() {
```

Comment:
Use other methods to compare, but is this by default private, or do you need a jsdoc comment?

Reply:
Added a jsdoc for this